ts/Extensions/OverlappingDataLabels.ts

```diff
@@ -89,7 +89,42 @@ function chartHideOverlappingLabels(
             box2.x + box2.width <= box1.x ||
             box2.y >= box1.y + box1.height ||
             box2.y + box2.height <= box1.y
-        );
+        ),
+        pointIsInPolygon = (p: Vec, polygon: Vec[]): boolean => {
```

Comment:
Later, we can import this function from https://github.com/highcharts/highcharts/blob/master/ts/Maps/MapUtilities.ts#L49-L81.